support-frontend/assets/helpers/redux/checkout/payment/existingPaymentMethods/thunks.ts

```diff
@@ -28,11 +29,24 @@ export const getExistingPaymentMethods = createAsyncThunk<
 		const mdapiUrl = getGlobal<string>('mdapiUrl');
 
 		if (mdapiUrl) {
+			const authWithOkta = isSwitchOn('featureSwitches.authenticateWithOkta');
+			const accessToken = cookie.get('GU_ACCESS_TOKEN');
+
+			// No point in making the call if we don't have an access token as we know it's going to fail
+			if (authWithOkta && !accessToken) return [];
+
+			// Okta authorization uses the Authorization header rather than a cookie
+			const authHeader =
+				authWithOkta && accessToken
```

Comment:
```suggestion
				authWithOkta 
```
I think this is the same isn't it? We've checked accessToken above

Reply:
It's redundant but I think if I didn't do this the linter complained, but I'll have another look.

Reply:
Removed check in a2e2445e95effdeb9f8a54fc90187148c3bd312d